packages/checkout/widgets-lib/src/resources/text/textConfig.ts

```diff
@@ -497,6 +497,20 @@ export const text = {
         heading: 'Choose a wallet to view',
       },
     },
+    [BridgeWidgetViews.CLAIM_WITHDRAWAL]: {
+      content: {
+        heading: 'Follow the prompts to receive your coins on ',
+        body: 'You’ll be asked to pay a small gas fee',
+        body2: 'Destination Wallet address: ',
+      },
+      footer: {
+        buttonText: 'Continue',
+        retryText: 'Try again',
+      },
+      loadingView: {
+        text: 'Initiating move',
+      },
+    },
```

Comment:
Let's remove this since Charlie has [this PR](https://github.com/immutable/ts-immutable-sdk/pull/1427) up to remove textConfig

Reply:
good call